### ISSUE TITLE
Variable names are red

### ISSUE THREAD

**Issue by juaninicolai** [one-dark/jetbrains-one-dark-theme#321]
Hello,

I really like your theme. Thank you for that.
Would it be possible to change the source code so that variables are not in red color? It's misleading since the default theme or almost every editor highlight errors as red.

Thanks, Juan

**Comment by alwibrm** [comment]
I agree, this is very irritating. Please choose another colour for fields.
